examples/boxplot.rs

```diff
@@ -94,13 +93,14 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
                     .offset(*offset)
             }))?
             .label(label)
-            .legend(move |(x, y)| Rectangle::new([(x - 5, y - 5), (x + 5, y + 5)], style.filled()));
+            .legend(move |(x, y)| Rectangle::new([(x, y - 3), (x + 12, y + 9)], style.filled()));
```

Comment:
Legend is drawn correctly, I've just changed the rectangle location to be aligned more with the text (original change had a misalignment already, but I hoped that it could be fixed with the proper text rendering, but as I mentioned above, the legends rendering is not easy to fix and out of scope of this PR).